scripts/set-up-wordpress-dependencies.sh

```diff
@@ -27,8 +27,8 @@ mv "${RUNNER_TEMP}/plugin-loader.php" "${WP_CONTENT_TARGET_DIR}/client-mu-plugin
 git_checkout "${WP_CONTENT_TARGET_DIR}/mu-plugins" https://github.com/Automattic/vip-go-mu-plugins-built.git 1
 
 # Install memcached drop-in.
-if [[ ! -f "${WP_CONTENT_TARGET_DIR}/object-cache.php" && -f "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/object-cache/object-cache-stable.php" ]]; then
-  ln -s "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/object-cache/object-cache-stable.php" "${WP_CONTENT_TARGET_DIR}/object-cache.php"
+if [[ ! -f "${WP_CONTENT_TARGET_DIR}/object-cache.php" && -f "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/wp-memcached/object-cache.php" ]]; then
+  ln -s "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/wp-memcached/object-cache.php" "${WP_CONTENT_TARGET_DIR}/object-cache.php"
```

Comment:
We should point to the `mu-plugins/drop-ins/object-cache.php` file rather than reference a specific drop-in. That way, hopefully, we don't have to make changes like this if VIP updates their drop-in in the future.